docs/ERROR_HANDLING.md

```diff
@@ -0,0 +1,70 @@
+# Stark Error Handling
+
+Stark provides its own error handler, StarkErrorHandler, that overrides Angular's ErrorHandler. 
```

Comment:
"... that overrides Angular's default ErrorHandler" .... and make this a link to this page: https://angular.io/api/core/ErrorHandler